### DISCUSSION TITLE
3D Files

### DISCUSSION THREAD

**Discussion by JesseK1991** [probonopd/MiniDexed#260] [Q&A] [answered]
Hello everyone, just yesterday I was able to put everything together and with the DAC the audio quality has a good improvement. 
I am surprised by the community of this project, thank you very much to all the collaborators who have contributed ideas to this project.
I was browsing the wiki for this project and found a 3d model of the case;
<img width="736" alt="Captura de Pantalla 2022-05-27 a la(s) 1 38 56 p m" src="https://user-images.githubusercontent.com/106403788/170776137-b99eb70d-104d-4007-a335-ecbf1dab58db.png">
But I did not find anywhere on this site the files to print it. Are the files accessible to everyone or is someone selling them?
Thank you very much greetings

**Comment by chemati8** [comment]
https://www.thingiverse.com/thing:5371115. 
You can download this case for printing

**Comment by probonopd** [comment, marked as answer]
> Are the files accessible to everyone or is someone selling them?

You can now download the STL from the wiki page. have fun!
